PRC.owl

```diff
@@ -301,7 +301,7 @@
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/PRC.owl/PRC_0000002">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/PRC.owl/PRC_0000001"/>
-        <obo:IAO_0000115>A protein conformation that is realized in the formation of hydrogen bonds between the partially negative oxygen atom and partially positive nitrogen atom in the polypeptide backbone of amino acids and the specific 3-dimensional structure that results.</obo:IAO_0000115>
+        <obo:IAO_0000115>A protein conformation that is realized in the formation of hydrogen bonds between the partially negative oxygen atoms and partially positive nitrogen atoms in one or more polypeptide backbones of amino acids and the specific 3-dimensional structure that results. Beta sheets in particular may involve the same or separate amino acid chains, whether most other secondary structures involve only a single amino acid chain.</obo:IAO_0000115>
```

Comment:
Change the "one or more polypeptide backbones of amino acids" to "one or more amino acid chains" as this is more consistent with usage in the Protein Ontology and PRC.

Change the word "whether" to "whereas".

Reply:
Also add your ORCID and name as dc:creator.